sdks/wasm/ProxyDriver/TestHarnessStartup.cs

```diff
@@ -147,6 +147,11 @@ public void Configure (IApplicationBuilder app, IOptionsMonitor<TestHarnessOptio
 						if (!str.StartsWith ("DevTools listening on ", StringComparison.Ordinal))
 							return null;
 
+						// Unfortunately it does look like we have to wait
+						// for a bit after getting the response but before
+						// making the list request
```

Comment:
Do we know why? Is there a tracking issue for this that the comment can reference, or a way to describe what happens if we don't wait long enough? If it needs to actually be say 700 or 900 it'd be good to have a comment here explaining what symptoms can be addressed by raising the delay

Reply:
I don't know why, I'm not sure that there is a strict contract there to begin with.  I've refactored the request code to use a shorter delay but retry for up to two seconds, and added a more informative message when it fails.  There is definitely room for more improvement in this code but I'd like to get the harness running asap.